shared-libs/message-utils/test/index.js

```diff
@@ -205,7 +207,8 @@ describe('messageUtils', () => {
       });
 
       it('integer', () => {
-        const date = 1457235941000;
+        // Date integer in running timezone
+        const date = moment(dateStr(1457235941000)).unix()*1000;
```

Comment:
I think this really reduces the quality of our testing because you're stripping off the time component. I take it this test was failing in your tz?

It really shouldn't be because both the util and the test are using moment in exactly the same way, right? Can you explain why the test is failing for you?